src/inline-contracts/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ovh_inline_contracts_label_unique": "Je reconnais avoir bien pris connaissance des {{ contract }}.",
+  "ovh_inline_contracts_label_first": "Je reconnais avoir bien pris connaissance des {{ contract }},",
+  "ovh_inline_contracts_label_next": "et des {{ contract }},",
```

Comment:
Pretty sure this doesn't work for all cases and for all languages 😅

Reply:
As it is a component and will be reused, it will be easier to iterate and improve :)